editor/modes/visual-editor/style.scss

```diff
@@ -65,6 +65,34 @@
 		transition: 0.2s outline;
 	}
 
+	&.is-showing-mobile-controls {
+		.editor-visual-editor__block-controls > div {
```

Comment:
Relies on slot behavior wrapping with `div`. Might be less fragile by targeting anything _except_ mobile tools:

```css
.editor-visual-editor__block-controls > :not( .editor-visual-editor__mobile-tools ) {
```